test/mocks/GemMock.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.13;
+
+contract GemMock {
```

Comment:
We can just import MockERC20 for this: https://github.com/marsfoundation/erc20-helpers/blob/9cf80a0d21692a4442c38bf2cb76c866ee3b7d85/src/MockERC20.sol

Reply:
Done.